packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -419,17 +430,17 @@ const systemPage = (
         <Grid item md={6} xs={12}>
           <EntityCatalogGraphCard variant="gridItem" height={400} />
         </Grid>
-        <Grid item md={4} xs={12}>
-          <EntityLinksCard />
+        <Grid item md={4}>
+          <SecurityChampionCard />
         </Grid>
         <Grid item md={8}>
           <EntityHasComponentsCard variant="gridItem" />
         </Grid>
         <Grid item md={6}>
           <EntityHasApisCard variant="gridItem" />
         </Grid>
-        <Grid item md={6}>
-          <EntityHasResourcesCard variant="gridItem" />
```

Comment:
whats the consequence of removing this?

Reply:
It seems like EntityHasResourcesCard is not used so I removed it to reduce noise in the UI. 

We can put it on the bottom instead of removing it.